base/dcp_test.go

```diff
@@ -180,15 +180,16 @@ func TestCBGTIndexCreation(t *testing.T) {
 				}
 
 				err = context.Manager.CreateIndex(
-					"couchbase",      // sourceType
-					bucket.GetName(), // sourceName
-					bucketUUID,       // sourceUUID
-					sourceParams,     // sourceParams
-					indexType,        // indexType
-					legacyIndexName,  // indexName
-					indexParams,      // indexParams
-					planParams,       // planParams
-					"",               // prevIndexUUID
+					// TODO: this should be cbgt.SOURCE_GOCOUCHBASE but this causes test setup to fail
```

Comment:
Should we be keeping the test for the old approach around, and only run with older CBS versions?  Even as a disabled, dev-only test.

Reply:
This is maybe a more general question about our integration testing - with all the changes for collections, I suspect we should think about having an option to run the integration tests against a 6.5 server to catch regressions.

Reply:
We already have the capability - I've filed CBG-2216 to look into running it on a regular basis.